core/src/main/scala/org/http4s/headers/Sec-WebSocket-Accept.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package headers
+
+import org.typelevel.ci._
+import org.http4s.internal.parsing.Rfc7230
+
+final case class `Sec-WebSocket-Accept`(accept: String)
```

Comment:
This public constructor lets us construct values from non-tokens and circumvent the parser.

Reply:
Passing in a byte array here now, but removing the `case` modifier since `Array`s can't be compared by value. Maybe we can use `ByteVector` or a custom `ByteArray` type if it doesn't exist alread